.github/workflows/cache-test-images.yaml

```diff
@@ -0,0 +1,80 @@
+name: Cache test images
+on:
+  workflow_dispatch:
```

Comment:
I'd like to add `schedule`.

Reply:
do this job once a day?

Reply:
Yes, let's start with daily. Most images are unchanged, and we don't call GHCR API other than listing tags.

Reply:
Done https://github.com/aquasecurity/trivy/pull/7614/commits/91a8016b42a752ace91dd904b194d56005931bb4